monitoring/ingestion/ingestion-producer-alert.yaml

```diff
@@ -0,0 +1,74 @@
+x-inputs:
+  - name: namespace
+    type: constant
+  - name: job_ingestion_producer
+    type: constant
+  - name: replicas
+    type: constant
+
+groups:
+- name: IngestionProducer
+  rules:
+
+  - alert: IngestionProducerDown
+    Expr: sum(up{namespace="${namespace}", job="${job_ingestion_producer}"}) == 0
+    For:   "30s"
+    Labels:
+      severity: critical
+    Annotations:
+      zenko_service: backbeat-ingestion-producer
+      description: "Ingestion Producer pod has been down for 30 seconds"
+      summary: "Ingestion Producer service is down"
+
+  - alert: IngestionProducerDegraded
+    Expr: sum(up{namespace="${namespace}", job="${job_ingestion_producer}"}) < ${replicas}
+    For:   "30s"
+    Labels:
+      severity: warning
+    Annotations:
+      zenko_service: backbeat-ingestion-producer
+      description: "Ingestion Producer pod has been down for 30 seconds"
+      summary: "Ingestion Producer service is degraded"
+
+  - alert: IngestionProducerLagIncreasedBy3Percent
+    Expr: |
+      ((sum by (origin) (metadata_journal_size-ingestion_read_offset+1) -
```

Comment:
What we do is specifying the job label to gather metrics from the same job. 
The job label name is the name of the Kubernetes services (e.g: dev-backbeat-ingestion-producer-headless).
The job label is set by Prometheus operator when the Service Monitor discovers the Service and its endpoints.
If we use job label, we won't need to sum by origin.

We also specify the namespace, in case one day we deal with metrics coming from different namespaces.